src/patch-evaluation-service/config/dev-config.json

```
@@ -0,0 +1,15 @@
+{
+    "version": "0.1.0",
```

```suggestion
    "version": "0.2.3-alpha",
```
suggest aligning with the current release version